checkov/helm/runner.py

```diff
@@ -89,6 +96,31 @@ def run(
             # TODO: Export helm dependencies for the chart we've extracted in chart_dependencies
             return report
 
+    def get_image_report(self, root_folder: str | None, runner_filter: RunnerFilter) -> Report | None:
+        if not self.graph_manager:
+            return None
+        return self.check_container_image_references(
+            graph_connector=self.graph_manager.get_reader_endpoint(),
+            root_path=self.original_root_dir,
+            runner_filter=runner_filter,
+        )
+
+    def extract_images(
+            self,
+            graph_connector: DiGraph | None = None,
+            definitions: None = None,
+            definitions_raw: dict[str, list[tuple[int, str]]] | None = None
+    ) -> list[Image]:
+        if not graph_connector:
```

Comment:
can't we change the type annotation to not accept `None` here and then remove the condition?